app/metrics.yaml

```diff
@@ -83,3 +83,168 @@ searches:
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_out:
+    type: event
+    description: >
+      A user pressed the sign out button on the sync account page and was successfully signed out of FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  bookmarks_sync_status:
+    type: boolean
+    description: >
+      Status of the bookmarks sync engine
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  history_sync_status:
+    type: boolean
+    description: >
+      Status of the history sync engine
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sent_tab_success:
+    type: event
+    description: >
+      A user successfully sends a tab
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sent_tab_error:
+    type: event
+    description: >
+      A user gets an error while sending a tab
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  received_tab:
+    type: labeled_counter
+    description: >
+      Number of received tabs
+    labels:
+      - desktop
+      - mobile
+      - tablet
+      - tv
+      - vr
+      - unknown
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
```

Comment:
nit: newline missing at the end of the file

Reply:
Added